collect_app/src/main/java/org/odk/collect/android/activities/GeoPointMapActivity.java

```diff
@@ -581,12 +547,11 @@ public String getLocationStatus() {
     }
 
     public String getAccuracyStringForLocation(Location location) {
-        return getString(R.string.location_provider_accuracy, GeoPointUtils.capitalizeGps(location.getProvider()),
```

Comment:
This call was desired to make gps -> GPS. Is there a reason you've removed it? If you do change behavior, please make sure that there's an explanation somewhere (commit messages ideally or maybe PR description).

Reply:
Same as above.